Contributing.md

```diff
@@ -56,6 +56,70 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the TC for resolution and that discussion and
 compromise among committers be the default resolution mechanism.
 
+# Triaging
+
+Open issues in this repo are subject to the following triage criteria:
+
+`needs triage`: Issues that are opened and un-attended / unclassified belong to this 
+category, and are either automatically or manually labelled with `needs triage`
+
+`question`: Issues that are not leading upto a bug or an enhancement to any of the 
+express repos, but emanating from the usage of it are classified as `question`
+
+`bug`: Issues that, after investigation, provides a reasonable conviction about
+a potential bug in the express code, are classified as `bug`
+
+`enhancement`: Issues that, after discussion, investigation, and technical assessment,
+found to be a reasonable candidate for a good to have feature, are tagged as `enhancement`.
+
+Raisers may appeal to change the tag with sufficient reasoning.
+Maintainers can change the tag based on the change in inference in the course of investigation.
+
+In all cases, issues may be closed by maintainers if they don't receive timely response when
+further information is sought, or when additional questions are asked.
+
+## Approaches and Best Practices for getting into triage contributions
+
+ - review the organization's [Dashboard][]
```

Comment:
```suggestion
 - review the organization's [StatusBoard](https://expressjs.github.io/statusboard/)
```

Reply:
I think this section would read better as a paragraph than a series of bullet points:


> Review the organization's [StatusBoard](https://expressjs.github.io/statusboard/), pay special attention to these columns: stars, watchers, open issues and contributors. This gives you a general idea about the criticality and health of the repository. Pick a few projects based on that criteria and your interests and skills (existing or aspiring).
> 
> Review the project's contribution guideline if present. In a nutshell, pledge yourself to the community's standards and values. Review the documentation, for most of the projects it is just the `README.md`, and make sure you understand the key APIs, semantics, configurations, and use cases.
> 
> It might be helpful to write your own test programs to re-affirm your understanding of the key functions. This may identify some gaps in documentation, record those as they might be good PR's to open later. Skim through the issue backlog; identify low hanging ones and mostly new ones. From those, attempt to recreate issues based on the OP description and sk questions if required. No question is a bad question!

The rest of the points I think do fit as a bullet list, but maybe put a header above them like "Other Helpful Hints"?

Reply:
@wesleytodd - thanks for the suggestion; I followed this model